test.py

```diff
@@ -89,6 +89,48 @@
 
 easy_spanish_text = "Hoy es un lindo día"
 
+long_russian_text_guillemets = (
+    "Игра ... игры всегда считались важными для развития "
+    "уравновешенных и творческих детей; однако, какую роль "
+    "они должны играть в жизни взрослых, если таковая имеется, "
+    "никогда не исследовалась так глубоко. Я считаю, "
+    "что игры для взрослых не менее важны, чем для детей. "
+    "Выделение времени для игр с нашими детьми и другими "
+    "взрослыми не только ценно для построения межличностных "
+    "отношений, но также является прекрасным способом снять "
+    "накопившееся напряжение.\n"
+    "Ничто не доставляет такого же удовольствие для моего мужа "
+    "после тяжелого рабочего дня, как прийти домой и поиграть "
+    "с кем-нибудь в шахматы. Это позволяет ему расслабиться от "
+    "повседневных дел и обсудить плюсы и минусы дня в спокойной "
+    "обстановке. Один из самых запоминающихся свадебных "
+    "подарков - набор нардов - получил близкий друг. Я спросила "
+    "его, зачем он сделал нам такой подарок. Он ответил, что "
+    "считает важным аспектом брака никогда не прекращать "
+    "совместные игры. По прошествии лет, когда я "
+    "начала покупать и проходить с другими парами и коллегами "
+    "многие игры, такие как: Monopoly, Chutes & Ladders, "
+    "Mastermind, Dweebs, Geeks, & Weirdos и т.д. Я сознаю их "
+    "неотъемлемую роль, которую они сыграли в наши выходные и "
+    "в наши недели аля «выключите телевизор и займитесь "
+    "чем-нибудь более стимулирующим». Они обогатили мою "
+    "жизнь и сделали ее интереснее. К сожалению, многие "
+    "взрослые забывают, что игры вообще существуют, а "
+    "прячут их в шкафы, о которых забывают, пока не придут "
+    "внуки.\n"
+    "Слишком часто взрослые настолько увлечены работой, чтобы "
+    "платить по счетам и не отставать от «Джонсов», что "
+    "пренебрегают радостью жизни; удовольствием, которое может "
+    "быть наградой за расслабляющую игру с другим человеком. "
+    "Было сказано, что «человек - это для того, чтобы иметь "
+    "радость», но слишком часто мы идем по жизни без особой "
+    "радости. Игры позволяют нам расслабиться, узнать что-то "
+    "новое и интересное, взаимодействовать с людьми на другом, "
+    "более комфортном уровне и наслаждаться безопасным "
+    "соревнованием. По этим причинам взрослые должны уделять "
+    "больше внимания играм в своей жизни"
```

Comment:
The final sentence doesn't end with a full stop, is that right?

Reply:
That's right.  It's a translation of `long_test` with the only formatting change being the guillemets ( '«' and '»' ).
I messed up with the first commit, but it's fixed, and the test passed with my `set_lang` implementation.

Reply:
:+1: